src/presets/cloudflare.ts

```diff
@@ -46,12 +50,6 @@ const cloudflarePreset: Preset = {
         [`node:${p}`, `node:${p}`],
       ]),
     ),
-    // The `node:assert` implementation of workerd uses strict semantics by default
-    "assert/strict": "node:assert",
-    "node:assert/strict": "node:assert",
-    // The `node:sys` module is just a deprecated alias for `node:util` which we implemented using a hybrid polyfill
-    sys: "unenv/runtime/node/util/$cloudflare",
```

Comment:
@anonrig I think we already discussed that one and the conclusion was that we need it because not all functions are implemented. It should not be a node compat module as long as util is not. Could you please revert this?